src/main/java/seedu/flashcard/ui/FlashcardQuestionCard.java

```diff
@@ -70,7 +70,7 @@ public FlashcardQuestionCard(Flashcard flashcard) {
 
         if (flashcard.getRating().toString().length() > 0) {
             rating.setText(flashcard.getRating().toString());
-            ratingIcon.setText(" \uD83D\uDFCA");
+            ratingIcon.setText(" \u2605");
```

Comment:
You may consider creating an Icon class to store the commonly used Unicode characters

Reply:
done